packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -149,6 +153,20 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
 
       <Drawer.Body>
         <PanelGroup>
+          <Panel>
+            <ControlLabel>{t('authors.panels.linkedUsers')}</ControlLabel>
+            <TagGroup style={{marginTop: '0px'}}>
+              {users && users.length > 0 ? (
+                users.map(user => (
+                  <Tag color="blue" key={user?.id}>
```

Comment:
it's required, no need to use `?`

Reply:
The object can be possibly null

Reply:
if you update the GraphQL as i mentioned above the object can not be null anymore